### ISSUE TITLE
[Accessibility] Health and Stamina buttons should have labels

### ISSUE THREAD

**Issue by citelao** [filiph/edgehead#11]
If you add an `aria-label` to the buttons (which should be buttons within a list, rather than a list item that's a button), screen reader users will understand what the buttons mean.

The icons were confusing to me, as well, which suggests that they should have *visible* text representation, too.
